src/GraphQL.Tests/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Fixie" version="1.0.0.29" targetFramework="net45" userInstalled="true" />
+  <package id="Fixie" version="1.0.0.33" targetFramework="net45" userInstalled="true" />
```

Comment:
Upgraded from 1.0.0.29 to 1.0.0.33. This is not strictly needed, but thought it was worth being on par with the latest version of Fixie.